lm_eval/tasks/multiwoz/utils.py

```
@@ -0,0 +1,313 @@
+    return {
+        "service": str(frame.get("service", "")),
+        "state": {
+            "active_intent": str(state.get("active_intent", "NONE")),
+            "requested_slots": list(state.get("requested_slots", [])),
+            "slots_values": slot_values,
+        },
+        "slots": slots,
+    }
+
+
+def load_multiwoz_dataset(**kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)